cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -838,11 +850,18 @@ instance (Show txid, Show tx)
       , "agency" .= String (pack $ show stok)
       ]
 
+instance Show peerAddr => ToJSON (ConnectionId peerAddr) where
```

Comment:
We have `ToJSON` instances for `SockAddr` and `LocalAddress`.  I think we only need the two of them to get rid of `Show peerAddr` constraint.

Reply:
So adding `ToJSON peerAddr` constraint is probably better, no?

Reply:
Yes, using `ToJSON` in place of `Show` is better.

Reply:
Done!